querqy-core/src/main/java/querqy/model/BoostedTerm.java

```diff
@@ -0,0 +1,38 @@
+package querqy.model;
```

Comment:
I'm still mulling over whether this is the right place to keep the boost and whether it should be added to any query type (for example, what happens if we inject a multi-term synonym, or if we decide to allow RawQuery on the right hand side of synonym rules).